src/components/Review/ReviewSectionRowProgress.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Grid } from 'semantic-ui-react'
+import { ReviewSectionComponentProps } from '../../utils/types'
+import { SectionProgressBar } from './SectionProgress'
+import strings from '../../utils/constants'
+
+const ReviewSectionRowProgress: React.FC<ReviewSectionComponentProps> = ({
+  action,
+  section,
+  isAssignedToCurrentUser,
+}) => {
+  // Set default for SectionProgressBar
+  const reviewProgress = section.reviewProgress || {
+    totalReviewable: 0,
+    doneConform: 0,
+    doneNonConform: 0,
+  }
+
+  const getContent = () => {
+    switch (action) {
+      case 'canStartReview': {
+        if (isAssignedToCurrentUser) return null
+        return strings.NOT_STARTED
+      }
+      case 'canView': {
+        return <SectionProgressBar reviewProgress={reviewProgress} />
+      }
+      case 'canContinue': {
+        if (isAssignedToCurrentUser) return <SectionProgressBar reviewProgress={reviewProgress} />
+        return strings.IN_PROGRESS
```

Comment:
```suggestion
      case ReviewAction.canStartReview: {
        if (isAssignedToCurrentUser) return null
        return strings.NOT_STARTED
      }
      case ReviewAction.canView: {
        return <SectionProgressBar reviewProgress={reviewProgress} />
      }
      case ReviewAction.canContinue: {
        if (isAssignedToCurrentUser) return <SectionProgressBar reviewProgress={reviewProgress} />
        return strings.IN_PROGRESS
```

Also need to import `ReviewAction` from types

Reply:
9800aa94306142062dcde84cc7d889d0a2e88b2d, i've made a seperate commit as it was easier to keep track of compilation error and do a fix all at once